virt-v2v/scripts/rhel/run/network_config_util.sh

```diff
@@ -24,9 +28,19 @@ if [ -f "$UDEV_RULES_FILE" ]; then
     exit 0
 fi
 
+# Helper functions
+# ----------------
+
+# Get the v2v map file as list 
+v2v_list_as_envs() {
+    while IFS= read -r line; do
+        echo "$line"
+    done < "$V2V_MAP_FILE"
```

Comment:
I think this is equal to `cat $V2V_MAP_FILE` as we just echo every line without modifications.

In the original code I was modifying the line to convert them into ENV_KEY=VALUE format to then export them iwth `export`

Reply:
thanks ! replaced ...